README.md

```diff
@@ -115,36 +115,47 @@ integrates with the incremental cache, and will recognize proof structure.
 You can edit without fear inside a `Proof. ... Qed.`, the rest of the document
 won't be rechecked, unless the proof is completed.
 
-### Whole-Document Goal Display
+<img alt="Smart error recovery" height="286px" src="etc/img/lsp-errors.gif"/>
+
+Furthermore, you can leave bullets and focused goals unfinished, and `coq-lsp`
+will automatically admit them for you.
+
+### 🥅 Whole-Document Goal Display
 
 Press `Alt+Enter` (or `Cmd+Enter` in Mac) to show goals at point in a side
 panel.
 
 <img alt="Whole-Document Goal Display" height="286px" src="etc/img/lsp-goals.gif"/>
 
-### Markdown Support
+The panel will also include goals that you have given up or shelved. When
+multiple goals are present, you can see short synopses of them.
+
```

Comment:
Maybe add "the panel will also show info about the current open bullets and their goals"